metadata.json

```diff
@@ -53,31 +53,26 @@
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "8",
-        "9",
         "10"
       ]
     },
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "16.04",
         "18.04",
         "20.04"
       ]
     },
     {
       "operatingsystem": "SLES",
       "operatingsystemrelease": [
-        "12.1",
-        "12.2"
       ]
```

Comment:
I thought SLES still has support?

Reply:
According to https://www.suse.com/lifecycle 12.1 support ended on 2017-05-31 and 12.2 support ended on 2018-03-31.

Maybe we can adjust this list with `["12"]`?

Reply:
yes, we should switch to major versions only.

Reply:
Updated in #1127